### ISSUE TITLE
Fungal infection of hair

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9968]
**Preferred term label:**
Fungal infection of hair

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Black piedra

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
would this be a positive culture or microscopic test?

**Comment by MickeySegal** [comment]
Microscopic.

https://www.ncbi.nlm.nih.gov/books/NBK559024/
"Direct microscopic examination of infected hairs mounted on slides with 10% potassium hydroxide (KOH) reveals endothrix infection."

https://www.ncbi.nlm.nih.gov/books/NBK482477/
"Microscopically, these are seen as yeast cells which then produce hyaline hyphae that cutoff at the septa and those hyphal elements act as arthroconidia. Pseudohyphae and blastoconidia are also observed but no teleomorphic states. "

https://www.ncbi.nlm.nih.gov/books/NBK545178/
"DME of the crushed nodules under 10% KOH shows branched hyphae (4 to 8 microns) held together by a cement-like substance. The dematiaceous filamentous hyphae appear as chains of stout thick-walled cells resembling arthrospores. Visualization enhancement can be via lactophenol cotton blue and other colored dyes."

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001088
- label: Fungal hair infection
- parents: [Positive infectious agent test (HP:0430030)](https://hpo.jax.org/app/browse/term/HP:0430030)
- pmid: PMID:31424762|PMID:29494084|PMID:32644450
- def: Demonstration of fungi growing on hair by a method such as microscopy of hairs mounted on slides with 10% potassium hydroxide.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9968
- synonyms: Fungal infection of hair(NONE)
